src/main/steps/sequence.ts

```diff
@@ -385,4 +387,8 @@ export const sequence: Step[] = [
     url: JURISDICTION_CONNECTION_SUMMARY,
     getNextStep: () => YOUR_NAME,
   },
+  {
```

Comment:
I don't think these pages should go into the steps array, that's just for applicant 1

Reply:
Where would be the best place for them?